content/About Arduino/My Arduino Account/Enable-2-Step-Verification-for-your-Arduino-account.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Enable 2-Step Verification for your Arduino account"
+id: 360018131120
+---
+
+Securing your Arduino account is crucial to protect your projects and personal information. Enabling 2-Step Verification adds an extra layer of security, requiring both your password and a code generated by an authenticator app to access your account.
+
+Arduino supports 2-step Verification via authenticator software such as _Authy_, _Google Authenticator_, _Microsoft Authenticator_ or similar.
+
+---
```

Comment:
I have the following suggestion, to try more clearly establish:

* the method of 2-step verification (TOTP)
* the authenticator app requirement

```suggestion
By enabling 2-Step Verification for your Arduino account, you add an extra layer of security in case your password is stolen.

---

## About authenticator apps

To use 2-step Verification, you will need an authenticator or password manager app with time-based one-time password (TOTP) support.

These are some popular choices:

* 1Password
* Twilio Authy
* Google Authenticator
* Microsoft Authenticator
* LastPass

## Enable 2-Step Verification
```